lpa/lpa_windows.cpp

```diff
@@ -14,9 +14,13 @@
 #include <iostream>
 using namespace std;
 
+// handle2 is "have read/write permission" flag
+#define hasIOAccess handle2
+
 LPA::Process::Process(QObject * parent, intptr_t pidx) : QObject(parent) {
     pid = pidx;
-    handle = reinterpret_cast<intptr_t>(OpenProcess(PROCESS_QUERY_INFORMATION | PROCESS_VM_READ | PROCESS_VM_OPERATION | PROCESS_VM_WRITE, true, static_cast<DWORD>(pidx)));
+    handle = reinterpret_cast<intptr_t>(OpenProcess(PROCESS_QUERY_INFORMATION, true, static_cast<DWORD>(pidx)));
+	hasIOAccess = false;
```

Comment:
Not to be too nitpicky, but this is a tab. You might need to adjust your Qt Creator settings.